src/app/components/map/page.jsx

```
@@ -1,55 +1,22 @@
 import React from 'react';
-import { APIProvider,Map,AdvancedMarker,MapCameraChangedEvent,Pin } from '@vis.gl/react-google-maps';
+import { APIProvider,Map } from '@vis.gl/react-google-maps';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)